idp-service/src/main/java/io/mosip/idp/services/ClientManagementServiceImpl.java

```diff
@@ -42,15 +43,15 @@ public ClientDetailResponse createOIDCClient(ClientDetailCreateRequest clientDet
         }
 
         ClientDetail clientDetail = new ClientDetail();
-        clientDetail.setJwk(getJWKString(clientDetailCreateRequest.getJwk()));
         clientDetail.setId(clientDetailCreateRequest.getClientId());
+        clientDetail.setPublicKey(getJWKString(clientDetailCreateRequest.getPublicKey()));
         clientDetail.setName(clientDetailCreateRequest.getClientName());
-        clientDetail.setRpId(clientDetailCreateRequest.getRelayingPartyId());
+        clientDetail.setRpId(clientDetailCreateRequest.getRelyingPartyId());
         clientDetail.setLogoUri(clientDetailCreateRequest.getLogoUri());
         clientDetail.setRedirectUris(String.join(",", clientDetailCreateRequest.getRedirectUris()));
         clientDetail.setClaims(String.join(",", clientDetailCreateRequest.getUserClaims()));
         clientDetail.setAcrValues(String.join(",", clientDetailCreateRequest.getAuthContextRefs()));
-        clientDetail.setStatus(clientDetailCreateRequest.getStatus());
+        clientDetail.setStatus(Constants.CLIENT_ACTIVE_STATUS);
```

Comment:
why are we overriting the request status ?

Reply:
The assumption is that on create of the OIDC client, it's always ACTIVE. so we are not taking the status from the request. We have removed the status field in the ClientDetailCreateRequest DTO.

Reply:
ok So it has to be removed in the schema of stoplight documentation also, hope that is taken care.

Reply:
yes, removed.